src/CppUTest/SimpleString.cpp

```diff
@@ -653,6 +653,30 @@ SimpleString StringFromMaskedBits(unsigned long value, unsigned long mask, size_
     return result;
 }
 
+SimpleString StringFromOrdinalNumber(unsigned int number)
+{
+    unsigned int onesDigit = number % 10;
+
+    const char* suffix;
+    if (number >= 11 && number <= 13) {
+        // all ordinal numbers in the 10s use 'th'.
+        suffix = "th";
+
+    // all other ordinals follow the same pattern based on the
+    // ones digit: 1 = "1st", 2 = "2nd", 3 = "3rd", rest = "4th"
```

Comment:
Could you remove the comments? They are pretty obvious and keeping them there is no use. In general, in CppUTest no comments are preferred over useless ones.